uint32 frame_id
uint8[6] data
